src/diffusers/utils/testing_utils.py

```diff
@@ -300,6 +300,24 @@ def preprocess_image(image: PIL.Image, batch_size: int):
     return 2.0 * image - 1.0
 
 
+def export_to_gif(images: List[PIL.Image.Image], output_gif_path: str = None) -> str:
+    if output_gif_path is None:
+        output_gif_path = tempfile.TemporaryDirectory().name
+
+    output_gif_path = Path(output_gif_path)
+
+    for i, image in enumerate(images):
+        image[0].save(
+            output_gif_path,
```

Comment:
This will overwrite the same file when using more than 1 sequence. Alternatives:
- Pass a single sequence (as the original type hints expressed), remove the loop.
- Pass a list of file paths (and create the temporary path inside the loop).

Reply:
@patrickvonplaten I think we can just export one gif each time? i.e. remove the loop?

Reply:
Let's go with that.